packages/realm/src/Results.ts

```diff
@@ -75,6 +75,10 @@ export class Results<T = unknown> extends OrderedCollection<T> {
     return this.internal.size();
   }
 
+  description(): string {
```

Comment:
General question: is the `description` the `filtered` string?

Reply:
Yes, it is core's representation of the query. Some SDKs are not using RQL so it is a serialization of the query.